sys/coreenv/src/Config.hs

```diff
@@ -33,16 +36,22 @@ data SolverConfig = SolverConfig
 newtype SolverId = SolverId { solverId :: String }
   deriving (Eq, Ord, Show)
 
+newtype ParameterName = ParamName { getParamName :: String }
+  deriving (Eq, Ord, Show)
+
+newtype ParameterValue = ParamValue { getParamValue :: String }
+  deriving (Eq, Ord, Show)
+
 -- | TorXakis configuration options.
 data Config = Config
   { -- | Log all SMT commands?
     smtLog           :: Bool
     -- | Available solvers that can be chosen from.
-  , availableSolvers :: Map.Map SolverId SolverConfig
-  , selectedSolver   :: SolverId
+  , availableSolvers     :: Map.Map SolverId SolverConfig
+  , selectedSolver       :: SolverId
+  , configuredParameters :: [(ParameterName,ParameterValue)]
```

Comment:
We need some documentation on what `configuredParameters` are (and also we need to update the readme/user-manual-which-doesn't-exists-yet so that people are aware of this possibility).

Reply:
I thought the name was self-explaining...